spec/fixtures/files/datahashes/core/intake1.json

```diff
@@ -2,16 +2,16 @@
     "entryNumber": "IN2020.21",
     "entryDate": "2020-09-25",
     "entryReason": "consideration",
-    "entryMethod": "post;foundondoorstep",
+    "entryMethod": "Post;Found on doorstep",
     "returnDate": "2020-09-17",
     "currentOwnerPerson": "2020",
     "currentOwnerOrganization": "2021",
     "depositorPerson": "Troy",
     "depositorOrganization": "Ninja",
     "depositorsRequirements": "All needed;Met",
     "approvalGroup": "board of trustees;executive committee;collection committee",
-    "approvalIndividual": "John Allen;Tim Joes",
-    "approvalStatus": "not approved; not required",
+    "approvalIndividual": "John Allen;Tim Joes;",
+    "approvalStatus": "not approved;;not required",
```

Comment:
There are three iterations of this group. Added delimiters to specify that. Definitely needed for approvalStatus to indicate blank value in 2nd group. Possibly not necessary for approvalIndividual, but good to be consistent/explicit